### PR TITLE
feat: feature discovery updates for inventory chart

### PR DESCRIPTION
**Pull request by chainzero**

### CODE REVIEW

File: charts/akash-inventory-operator/templates/config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+version: v1
+cluster_storage:
+  - default
```
Comment (troian) [inline charts/akash-inventory-operator/templates/config.yaml]: lets make them configurable via values.yaml

Comment (chainzero) [reply charts/akash-inventory-operator/templates/config.yaml]: values.yaml and config.yaml have been updated to allow config by user

File: charts/akash-inventory-operator/templates/deployment.yaml [hunk 2]
```diff
@@ -1,43 +1,84 @@
...
+              scheme: HTTP
+            initialDelaySeconds: 5
+            periodSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /metrics/ready
+              port: api
+              scheme: HTTP
+            initialDelaySeconds: 5
+            periodSeconds: 5
```
Comment (troian) [inline charts/akash-inventory-operator/templates/deployment.yaml]: lets exclude both probes for now. the are causing issues

Comment (chainzero) [reply charts/akash-inventory-operator/templates/deployment.yaml]: Removed liveness and readiness probes from inventory operator

**Review by troian — APPROVED** [review]

**Pull request merged**